model/websocket_message.go

```diff
@@ -295,8 +303,16 @@ func (ev *WebSocketEvent) ToJSON() ([]byte, error) {
 func (ev *WebSocketEvent) Encode(enc *json.Encoder) error {
 	if ev.precomputedJSON != nil {
 		return enc.Encode(json.RawMessage(
-			fmt.Sprintf(`{"event": %s, "data": %s, "broadcast": %s, "seq": %d}`, ev.precomputedJSON.Event, ev.precomputedJSON.Data, ev.precomputedJSON.Broadcast, ev.sequence),
-		))
+			// Performance optimization
+			[]byte(`{"event": ` +
+				string(ev.precomputedJSON.Event) +
+				`, "data": ` +
+				string(ev.precomputedJSON.Data) +
+				`, "broadcast": ` +
+				string(ev.precomputedJSON.Broadcast) +
+				`, "seq": ` +
+				strconv.Itoa(int(ev.sequence)) +
+				`}`)))
```

Comment:
Should we turn this into a method/function to avoid the repetition above?

Reply:
Sure